numpy/random/tests/test_random.py

```diff
@@ -205,12 +208,13 @@ def test_repeatability(self):
         assert_(tgt[np.dtype(np.bool).name] == res)
 
 
-class TestRandomDist(TestCase):
+class TestRandomDist:
     # Make sure the random distribution returns the correct value for a
     # given seed
 
-    def setUp(self):
-        self.seed = 1234567890
+    @classmethod
```

Comment:
With a generative test, in case of a test failure, you know immediately which iteration failed; additionally even if that happens the following iterations of the loop are still executed (and additional failures can be reported too).